src/component/dashboard/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import uuid from 'uuid/v4';
+import autoBind from '../../utils';
+import NoteForm from '../noteForm';
+import NoteList from '../noteList/index';
+
+export default class Dashboard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      notes: [],
+      error: null,
+    };
+
+    autoBind.call(this, Dashboard);
+  }
+
+  handleAddNote(notes) {
+    if (notes.title === '') {
+      return this.setState({ error: true });
+    }
+    if (notes.content === '') {
+      return this.setState({ error: true });
+    }
+
+    notes.createdOn = new Date();
+
+    return this.setState((previousState) => {
+      return {
+        notes: [...previousState.notes, notes, { ...notes, id: uuid() }],
```

Comment:
changing this back to:

```js
notes: [...previousState.notes, notes]
``` 

fixes the duplicate note issue, but you'll have to assign an id similar to how you're assigning a `createdOn` date.